src/controllers/EnrollmentController.ts

```diff
@@ -0,0 +1,73 @@
+import { Request, Response} from 'express'
+import Enrollment from '../models/Enrollment'
+
+class EnrollmentController{
+    static async getAllEnrollments(req: Request, res: Response){
+        try{
+            const enrollments = await Enrollment.find().populate('course').populate('student')
+            res.json(enrollments)
+        }catch(error){
+            console.log(error)
+        }
+    }
+
+    static async getEnrollmentById(req: Request, res: Response){
+        const EnrollmentId = req.params.id
+        try{
+            const enrollment = await Enrollment.findById(EnrollmentId).populate('course').populate('student')
+            if(!enrollment){
+                return res.status(404).json({ message: 'Enrollment not found'})
+            }
+            res.json(enrollment)
+        }catch(error){
+            console.log(error)
```

Comment:
please replace all console.log with log4js library (replace it with logger.info('...'), logger.error('....'))